authors/apps/profiles/tests/test_follows.py

```diff
@@ -0,0 +1,102 @@
+import os
+from rest_framework.test import APIClient, APITestCase
+from django.urls import reverse
+from django.test import TestCase
+from rest_framework import status
+from authors.apps.authentication.messages import error_msg, success_msg
+
+class UserFollow(APITestCase):
```

Comment:
Nice work on the test but I would suggest you change the name of the class  to something like TestUserFollow or UserFollowTests

Reply:
Thank you